packages/utils/plugins-helpers/src/types.ts

```diff
@@ -543,6 +543,10 @@ export namespace Types {
      * For more details: https://graphql-code-generator.com/docs/config-reference/lifecycle-hooks
      */
     hooks?: Partial<LifecycleHooksDefinition>;
+    /**
+     * @description Alows to raise errors if any matched files are not valid GraphQL. Default: false.
```

Comment:
I wrote this description in reference to https://the-guild.dev/graphql/tools/docs/api/modules/loaders_code_file_src#type-declaration